projects/1.16/assets/tips/tips/lang/zh_cn.json

```
@@ -1 +1,72 @@
-{}
\ No newline at end of file
+{
+  "tips.title.tip": "提示",
+  "tips.tip.no_tips": "未加载提示，请检查配置。",
+  "tips.tip.undead_damage": "像骷髅这样的亡灵生物会被治疗药水伤害，被伤害药水治疗。",
+  "tips.tip.poison_immune": "僵尸、骷髅和蜘蛛都免疫中毒效果的伤害",
+  "tips.tip.witch_conversion": "被闪电击中的村民将变成女巫。",
+  "tips.tip.four_eggs": "抛出鸡蛋时，有极小概率生成4只小鸡。",
+  "tips.tip.nether_lava": "熔岩在下界流动得更快。",
+  "tips.tip.east": "太阳、月亮和星星都向东方移动。",
+  "tips.tip.swamp_slime": "史莱姆会在满月时的沼泽中生成。",
+  "tips.tip.deflect_ghast_fireballs": "攻击恶魂火球可以改变它的运动方向。",
+  "tips.tip.shulker_armor": "潜影贝在打开时会受到更多的伤害。",
+  "tips.tip.arthropods": "节肢杀手能够对蜘蛛、蠹虫、蜜蜂、末影螨造成额外伤害。",
+  "tips.tip.fossils": "在沼泽和沙漠下能够发现大型化石。",
+  "tips.tip.crouch_blocks": "潜行可以防止你从方块边缘跌落。",
+  "tips.tip.smoke_bees": "在蜂巢下方放置营火会使蜜蜂不被激怒。",
+  "tips.tip.lava_fuel": "熔岩桶可以作为熔炉的燃料使用。",
+  "tips.tip.wooden_tool_fuel": "不再使用的木制工具能够作为熔炉燃料使用。",
+  "tips.tip.scrap_nuggets": "金制与铁制的工具、盔甲能够冶炼成粒",
+  "tips.tip.piglin_gold": "猪灵是真的非常喜欢黄金。",
+  "tips.tip.dog_health": "狼的尾巴越低，它剩的生命值就越少。",
+  "tips.tip.dog_rotten_flesh": "你可以通过喂狼吃腐肉来恢复它的血量。",
+  "tips.tip.lure_animals": "你可以在主手/副手拿上动物喜欢的食物来引诱动物跟随你。",
+  "tips.tip.cooked_drops": "因火焰伤害死亡的动物有时会掉落熟食。",
+  "tips.tip.nether_sponge": "在下界放置的湿海绵会瞬间干燥。",
+  "tips.tip.boat_passenger": "船能搭载两个乘客。",
+  "tips.tip.shulker_golem": "根据游戏代码，潜影贝被视为一种傀儡。",
+  "tips.tip.note_block_instrument": "音符盒放出的乐器声取决于音符盒下方的方块材料。",
+  "tips.tip.zombie_armor": "即使僵尸没有穿戴任何护甲，它们也拥有两点基础护甲值。",
+  "tips.tip.undead_helmet": "戴着头盔的亡灵生物在白天不会燃烧。",
+  "tips.tip.iron_golem_heal": "铁傀儡可以用铁锭来治疗。",
+  "tips.tip.turtle_shell": "杀死海龟并不会掉鳞甲，但小海龟长大时会掉。",
+  "tips.tip.animal_growing": "你可以喂养小动物来促进它们的成长。",
+  "tips.tip.blaze_snowball": "烈焰人是唯一会受到雪球伤害的敌对生物。",
+  "tips.tip.biome_bonemeal": "在草方块上使用骨粉，会根据你所在的生物群系给你不同的花。",
+  "tips.tip.biome_fishing": "与在其他生物群系中钓鱼相比，在丛林中钓鱼可以给你不同的物品。",
+  "tips.tip.fishing_resources": "钓鱼是获得高等级的附魔书的好方法。",
+  "tips.tip.boat_jump": "通过在熔岩中放置一艘船，你可以制作一个临时平台。",
+  "tips.tip.creeper_cat": "苦力怕害怕猫，如果太靠近猫，苦力怕会逃跑。",
+  "tips.tip.water_grass": "水流可以快速破坏草并掉落种子。",
+  "tips.tip.gravity_torch": "如果你手速够快，可以通过放火把在受重力影响的方块下面来破坏一串的这些方块。",
```

If you quickly put a torch under blocks affected by gravity, they will break in chain.
```suggestion
  "tips.tip.gravity_torch": "如果你迅速把火把插到受重力影响的方块下方，你就会看到它们接连被破坏。",
```
```suggestion
  "tips.tip.gravity_torch": "如果你迅速把火把插到受重力影响的方块下方，它们会被接火把连破坏。",
```

---

```suggestion
  "tips.tip.gravity_torch": "如果你迅速把火把插到受重力影响的方块下方，它们会被火把接连破坏。",
```